source/crud/crud.rst

```diff
@@ -978,6 +978,17 @@ Basic
      * @see https://docs.mongodb.com/manual/reference/command/delete/
      */
     collation: Optional<Document>;
+
+    /**
+     * The index to use.
```

Comment:
Considering these are probably copy-pasted from the spec to drivers, I wonder if we can do a little better in our doc string here. For the [find command](https://docs.mongodb.com/manual/reference/command/find/#dbcmd.find), our docs say:

> Optional. Index specification. Specify either the index name as a string or the index key
> pattern. If specified, then the query system will only consider plans using the hinted index.

Reply:
Sure thing! I'll update all the instances of this option.